tests/_helpers.py

```diff
@@ -9,12 +9,18 @@
         InMemorySpanExporter,
     )
     from opentelemetry.trace.status import StatusCode
+    from opentelemetry.semconv.attributes import (
+       OTEL_SCOPE_NAME,
+       OTEL_SCOPE_VERSION,
```

Comment:
I don't think we need this import here. Remove it and directly use this import in test__open... file

Reply:
Nope, we do need it as a polyfill when imports fail, please see https://github.com/googleapis/python-spanner/blob/a941adb987ec6e2d1a1583a5aa1254908292e354/tests/unit/test__opentelemetry_tracing.py#L6-L15 and then also
<img width="563" alt="Screenshot 2024-09-13 at 5 03 39 AM" src="https://github.com/user-attachments/assets/9f60cad2-a197-40a9-b694-eec8ff6cb13f"> aka my comment here https://github.com/googleapis/python-spanner/pull/1199#discussion_r1759036705

Reply:
look at my comment below

Reply:
Done, thanks @harshachinta!